.github/ISSUE_TEMPLATE/SlicingCrash.yaml

```diff
@@ -35,7 +35,7 @@ body:
 - type: markdown
   attributes:
     value: |
-      We work hard on improving our slicing crashes. Our most recent release is 5.6.0.
+      We work hard on improving our slicing crashes. Our most recent release is 5.7.1.
```

Comment:
this is inside placeholder for 5.6.0

Reply:
![image](https://github.com/Ultimaker/Cura/assets/28710690/c1c3fed4-2ed5-486e-8ada-d73e69510328)
It's the text under where the user submits the version

Reply:
I mean we should also change the placeholder on line 32

Reply:
IMPO
Since the majority of crashes are from before 5.7 then I would suggest keeping it as is.
Users would need to type the value either way so it's not removing/adding a step I would say.